src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -1085,7 +1085,8 @@ trait Types
     override def baseTypeSeq: BaseTypeSeq = supertype.baseTypeSeq
     override def baseTypeSeqDepth: Depth = supertype.baseTypeSeqDepth
     override def baseClasses: List[Symbol] = supertype.baseClasses
-  }
+    override def boundSyms: Set[Symbol] = emptySymbolSet
```

Comment:
Cool! `boundSyms` already confused me not long ago (I don't remember the details).